bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -144,17 +144,17 @@ public interface VoiceManager {
      * @param ks the keyword spotting service to use or null to use the default service
      * @param stt the speech-to-text service to use or null to use the default service
      * @param tts the text-to-speech service to use or null to use the default service
-     * @param hli the human language text interpreter to use or null to use the default service
+     * @param hlis collection of human language text interpreters to use or null to use the default service
```

Comment:
A small sentence in the description to explain the principle of this collection would be appreciated.